src/components/create-vm-dialog/createVmDialog.jsx

```diff
@@ -1210,6 +1210,18 @@ export class CreateVmAction extends React.Component {
                     </span>
                 </Tooltip>
             );
+        else
+            createButton = (
+                <Tooltip id={this.props.mode + '-button-tooltip'}
+                         content={this.props.mode === "create"
+                             ? _("Create virtual machine from local installation medium, URL, PXE network boot, cloud base image, or download an OS")
```

Comment:
My "bad grammar nerve" tickles here. "Create VM from .. or download an OS" reads weird; perhaps that needs a semicolon. But either way this feels rather long, these poor translators. Could we shorten this a bit to perhaps: "Create VM from local or network installation medium"?

Also, these tooltips are sentences and should end in a period, but there might be some different conventions here.

Reply:
Yeah, the strings are quite long. I'll try to shorten them. I like your suggestion here.

```suggestion
                             ? _("Create VM from local or network installation medium")
```

Reply:
We spell out VM elsewhere; I wonder if this needs it or if VM is fine enough. Either way, here's the expanded form:

```suggestion
                             ? _("Create virtual machine from local or network installation medium")
```

Reply:
I wanted to make it consistent with "existing VM installation", and reduce the string as well. But of course I'm fine with spelling it out.